api/routes/users/get.js

```diff
@@ -10,7 +10,7 @@ module.exports = (app, { getAllUsers = ga, getUserByID = gu } = {}) => {
 
   app.get('/users/:id', can('view-users'), async (req, res, next) => {
     await getUserByID(req.params.id)
-      .then(user => (user ? res.send(user) : res.status(404).end()))
+      .then(user => (user ? res.send(user) : res.status(400).end()))
```

Comment:
This feels like it should have stayed as a 404

Reply:
Good point. It could be either right?

Reply:
If I read the code right, we sent a request to the DB for User with ID of 5 and the DB returned something falsey, that seems like 404.  If "can('view-users')" came back as false then I think 403 is the right answer.

Reply:
@tbolt @knollfear 
So the team consensus is if we are searching for something in the database and it's not found, we return a 404? If someone is missing required input or sends in incorrect input is that a 400 or a 422?